docs/cosmology/traits.rst

```diff
@@ -0,0 +1,54 @@
+.. _astropy-cosmology-traits:
+
+****************
+Cosmology Traits
+****************
+
+.. currentmodule:: astropy.cosmology.traits
+
+The ``traits`` module hosts various parts of cosmologies, such as the
+:class:`~astropy.cosmology.traits.ScaleFactor` or :class:`~astropy.cosmology.traits.TemperatureCMB`. These traits can be used to more easily
+construct custom cosmologies by combining different components.
+
+As a simple example, the :class:`~astropy.cosmology.traits.TemperatureCMB` trait
+provides the ``Tcmb0`` property and
```

Comment:
Add a link.

Reply:
Can't. This was part of the change necessary to get Sphinx to guild.
Thankfully if someone clicks on TemperatureCMB it renders correctly.
6d3d657

Reply:
Can't. This was part of the change necessary to get Sphinx to guild.
Thankfully if someone clicks on TemperatureCMB it renders correctly.
6d3d657

Reply:
Every week I learn something new about our API documentation machinery.

Reply:
Yeah. It was an act of desperation.